src/scss/index.scss

```diff
@@ -22,3 +22,17 @@
   width: 100%;
   max-width: 150px;
 }
+
+// IE11 fix to vertically align center box within header
+.header > .box {
```

Comment:
I think it would adversely affect Box components in Header components (in IE11) if we apply the fixed height across all grommet apps (because the min-height, not height is set to `$header-height` in _objects.header.scss (https://github.com/grommet/grommet/blob/master/src/scss/grommet-core/_objects.header.scss#L4). I believe we only want it in people-finder, since the search input is contained in a box that is shorter than the min-height set by default grommet header styles.